frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -98,13 +109,24 @@ class Chart extends Component<Props, State> {
     ) {
       const data = this.state.source === 'hourly' ? chartDataHourly : chartDataDaily;
       this.lineSeries.setData(data);
+      this.setFormattedData(data);
     }
   }
 
   private hasData = (): boolean => {
     return this.props.data.chartDataDaily && this.props.data.chartDataDaily.length > 0;
   };
 
+  private setFormattedData(data : ChartData) {
+    if (!this.formattedData)
+      this.formattedData = {} as FormattedData;
```

Comment:
don't you want to do this every time? reset to `{}`

Reply:
talked to @thisconnect, he will try to enable a linter for formatting, e.g. no space in `data :` (before colon), require curly braces, etc.

Reply:
fixed!